composer.json

```
@@ -26,20 +26,19 @@
     "pimcore/object-merger": "^3.0",
```

```suggestion
    "pimcore/object-merger": "^4.0",
```

as placeholder for the moment, potentially it could be 3.1, depends if there would be bc breaks.
Opened an issue as reminder https://github.com/pimcore/object-merger/issues/16

https://github.com/pimcore/object-merger/blob/a878c05953259904058e0d8792d7b938ed4213d5/composer.json#L14